plugins/chat/assets/javascripts/discourse/components/channel-title/index.gjs

```diff
@@ -0,0 +1,111 @@
+import Component from "@glimmer/component";
```

Comment:
This is how I will set every components, I wanted to experiment on real use case before I generalize to other components.

Reply:
I'm kind of confused by this file and this comment; I only see `ChatChannelTitle` used in `OldChatChannelTitle` but then `import ChannelTitle from "discourse/plugins/chat/discourse/components/channel-title";` doesn't seem to exist anymore? :sweat_smile: Am I just missing some Ember trickery here?

Reply:
components/channel-title/index.gjs will l
load automatically when importing  components/channel-title